view/frontend/templates/js/replacejs.phtml

```diff
@@ -587,7 +587,7 @@ ob_start();
             }
         };
 
-        ////////////////////////////////////
+        if (getCheckoutType()=="payment") {
```

Comment:
Pre-fetch should be called on multi step checkout, not payment only.

Reply:
How I can emulate this mode on my local site?

Reply:
@vitaliyreznikov do we support payment only checkout? Was it ever used? I thought we should remove it.

Reply:
I asked @Abhinay-bolt and @usmanismail-bolt and as I understand we decided to save payment only for now.

About pre-fetch. Do I understand correctly we use it only to save shipping calculation in cache and get it from cache on shipping&tax call?

Reply:
Fixed